### ISSUE TITLE
Fix column filter error

### ISSUE THREAD

**Issue by kcajmagic** [xmidt-org/codex-db#8]
The table is defined in the README.md. `type` is not a column in the database. In fact it is a keyword for cassandra. (It is not reserved, which is why we didn't see the problem)
This need to be fixed here:
https://github.com/xmidt-org/codex-db/blob/dbee34f0e67db6d62310ba5f0d08b7586537c98c/cassandra/db.go#L160
suggestion:
```
 deviceInfo, err := c.finder.findRecords(limit, "WHERE device_id = ? AND record_type = ?", deviceID, eventType) 
```

If anywhere else the `type` column should be changed to `record_type`.

**Comment by iDevoid** [comment]
Hi @kcajmagic 
I am first timer to code on open-source, can i try to do this?
I also found another line that has `type = ?`. is this related?

https://github.com/xmidt-org/codex-db/blob/dbee34f0e67db6d62310ba5f0d08b7586537c98c/postgresql/db.go#L271

**Comment by kcajmagic** [comment]
@iDevoid this is a perfect issue for new timers. 

Yes it is related as `type` is also a keyword for Postgres. There will probably be more spots in that folder.

**Comment by iDevoid** [comment]
Hi @kcajmagic 
would you mind to take a look to my PR #10 ?
please tell me your opinion and advice. Thank you :pray:

**Comment by kcajmagic** [comment]
Fixed in PR #10
